client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -2030,6 +2030,15 @@ Undocumented.prototype.googleAppsFilterBySiteId = function( siteId, fn ) {
 	return this.wpcom.req.get( { path: '/sites/' + siteId + '/google-apps' }, fn );
 };
 
+Undocumented.prototype.isSiteImportable = function( site_url ) {
+	debug( `/wpcom/v2/site-importer-global/is-site-importable?${ site_url }` );
```

Comment:
The endpoint currently takes the param as `is-site-importable?site_url=https://xxxxxxx`. I'm not sure if that matters too much here, but it might end up confusing somebody.

Reply:
Good point! I think this made it's way in through some bad rebasing, as this wasn't meant to be part of this changeset. I've just properly rebased though so this should be sorted.
As for it being confusing - I agree! Let's make sure it's sorted before `add/signup-import-landing-handler` makes it to master. I'd guess that these debug statements will be stripped out by that point anyway.